### PR TITLE
Add libpostal integration milestone

### PR DESCRIPTION
**Pull request by dianashk**

### CODE REVIEW

File: Libpostal_Integration/README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Heavily depending on the [libpostal](https://github.com/openvenues/libpostal) mo
 
 ## Documentation
 
-Add a brand new documentation page devoted to the fallback mechanism. This is a new feature and significantly impacts results. Sweet all existing documentation to ensure all result examples are up-to-date, as many will have drastically changed for the better. There will be times when only a single result will be returned.
+Add a brand new documentation page devoted to the fallback mechanism. This is a new feature and significantly impacts results. Sweet *(what is this?)* all existing documentation to ensure all result examples are up-to-date, as many will have drastically changed for the better. There will be times when only a single result will be returned.
```
Comment (dianashk) [inline Libpostal_Integration/README.md]: Haha that's supposed to be sweep

**Comment by orangejulius** [comment]
LGTM

**Comment by trescube** [comment]
:shipit:

**Comment by dianashk** [comment]
LGTM (will this work? 😬)

**Pull request merged**